worlds/Files.py

```diff
@@ -41,7 +41,7 @@ def get_handler(file: str) -> Optional[AutoPatchRegister]:
 
 class AutoPatchExtensionRegister(abc.ABCMeta):
     extension_types: ClassVar[Dict[str, AutoPatchExtensionRegister]] = {}
-    required_extensions: List[str] = []
+    required_extensions: ClassVar[Tuple[str]] = ()
```

Comment:
This should be`Tuple[str, ...]` for a tuple of varying length who's members are all str, or `Iterable[str]` for any object that can be iterated over where the iteration returns str.

(Tuple without ellipsis has to have exactly the members it lists. I would recommend running mypy locally to find those mistakes before pushing to git.)